### PR TITLE
Added instructions to setup git in windows

### PR DESCRIPTION
**Pull request by vijaysaiy**

### CODE REVIEW

File: session-02-big-picture/README.md [hunk 1]
```diff
@@ -81,9 +81,27 @@ Steps to get PostgresDB installed locally
...
+5. Installation window will open, click on Next and you will be asked to choose installation directory, choose it and click on next.
+   ![](assets/install_window_1.png)
+   ![](assets/install_window_2.png)
+6. For the next upcoming screens just choose the default options and click next and wait for the installation to complete.
+7. After the installation is finished, click on Finish.
+8. To verify if git was properly installed, open terminal/command prompt/git bash and type
+
+   ```git --version```
+9. You should see something like this.
+   ![](assets/git_install_verify.png)
```
Comment (aliahmadcse) [inline session-02-big-picture/README.md:102]: Before making the clone, as part of the git setup, I think, we should set the global username and email for git with the commands:
`git config --global user.name "yourUsername"`
`git config --global user.email "yourEmail"`

Comment (koushikkothagal) [reply session-02-big-picture/README.md:102]: +1. It helps to have a section post-install. There are a couple of Windows-specific things to be done here.
Check this for reference: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup

File: session-02-big-picture/README.md [hunk 2]
```diff
@@ -81,9 +81,27 @@ Steps to get PostgresDB installed locally
 Possibly prerecorded videos. Need offline support
 
 # Git and GitHub
-TODO
-Steps to get Git installed
-Possibly prerecorded videos. Need offline support
+1. Go to download page using the following link
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:84]:
```suggestion
## On Windows
1. Go to download page using the following link
```

File: session-02-big-picture/README.md [hunk 3]
```diff
@@ -81,9 +81,27 @@ Steps to get PostgresDB installed locally
...
+
+   ```https://git-scm.com/downloads```
```
Comment (koushikkothagal) [inline session-02-big-picture/README.md:86]:
```suggestion
   [https://git-scm.com/downloads](https://git-scm.com/downloads)
```

**Review by koushikkothagal — CHANGES_REQUESTED** [review]
Thanks for the PR! Appreciate the detail in the screenshots (with callouts).
Please address the minor suggestions and we should be good to go!

File: session-02-big-picture/README.md (hunk 1 above)
Comment (vijaysaiy) [reply session-02-big-picture/README.md:103]: Addressed the suggestions, please verify and let me know if any further changes are required. Thank you

**Review by koushikkothagal — APPROVED** [review]

**Pull request merged**